ipxe/ipxe/build.sh

```diff
@@ -19,6 +19,10 @@ ipxe.efi)
 	cp $topdir/src/config/local/general.efi.h $topdir/src/config/local/general.h
 	make -C $topdir/src VERSION_PATCH=255 EXTRAVERSION="+ ($short_version)" bin-x86_64-efi/ipxe.efi
 	;;
+snponly.efi)
+	cp $topdir/src/config/local/general.efi.h $topdir/src/config/local/general.h
+	make -C $topdir/src VERSION_PATCH=255 EXTRAVERSION="+ ($short_version)" bin-x86_64-efi/snponly.efi
+	;;
 snp.efi)
 	cp $topdir/src/config/local/general.aarch64-snp-nolacp.h $topdir/src/config/local/general.h
 	# http://lists.ipxe.org/pipermail/ipxe-devel/2018-August/006254.html
```

Comment:
I didn't check this in.. but below I had to change:
```
CROSS_COMPILE=aarch64-unknown-linux-gnu-
```
to
```
CROSS_COMPILE=aarch64-linux-gnu-
```

Reply:
nix-shell would get you the correct compiler. I'll add this to docs.